client/my-sites/stats/subscribers-section/index.tsx

```diff
@@ -50,15 +61,40 @@ export default function SubscribersSection( { siteId }: { siteId: string } ) {
 
 	const chartData = transformData( data?.data || [] );
 
+	const subscribers = {
+		label: 'Subscribers',
+		path: `/stats/subscribers/`,
+	};
+
+	const date = new Date();
+
+	const slugPath = slug ? `/${ slug }` : '';
+	const pathTemplate = `${ subscribers.path }{{ interval }}${ slugPath }`;
+
 	return (
 		<div className="subscribers-section">
 			<div className="subscribers-section-heading">
 				<h1 className="highlight-cards-heading">Subscribers</h1>
+				{ ! isLoading && (
+					<div>
+						<StatsPeriodHeader>
+							<StatsPeriodNavigation date={ date } period={ period } url={ pathTemplate }>
+								<DatePicker
+									period={ period }
+									date={ date }
+									statsType="statsTopPosts"
```

Comment:
Can we rename or remove `statsTopPosts`, or does it break something at the moment?